apps/wing/src/commands/compile.ts

```diff
@@ -59,6 +61,23 @@ function resolveSynthDir(outDir: string, entrypoint: string, target: Target) {
   return join(outDir, `${entrypointName}.${targetDirSuffix}`);
 }
 
+// from diagnostic.rs
+interface Error {
+  message: string;
+  span: {
+    start: {
+      line: number;
+      col: number;
+    };
+    end: {
+      line: number;
+      col: number;
+    };
+    file_id: string;
+  };
+  level: "Error" | "Warning" | "Note";
+}
```

Comment:
This type should go in src/wingc.tc to help convey that it's sourced from there. Also it should probably be called `Diagnostic` or something

Reply:
Done